composite/dns-record-check/src/adapter.ts

```diff
@@ -0,0 +1,33 @@
+import { ExecuteWithConfig, ExecuteFactory, Config } from '@chainlink/types'
+import { Validator } from '@chainlink/external-adapter'
+import { Requester } from '@chainlink/external-adapter'
+import DNS from '@chainlink/dns-query'
+import { DNSResponse, DNSResponseAnswer } from '@chainlink/dns-query/dist/types'
+import { makeConfig } from './config'
+
+const inputParams = {
+  record: true,
+}
+
+const execute: ExecuteWithConfig = async (input, config) => {
```

Comment:
Do we want to use anything from config here?

Reply:
We need the config to set up the underlying `dns-query` adapter, so just pass it on to it if this adapter does not have any use for it.